pandas/tests/groupby/methods/test_describe.py

```diff
@@ -88,22 +88,24 @@ def test_frame_describe_multikey(tsframe):
     tm.assert_frame_equal(result, expected)
 
 
-def test_frame_describe_tupleindex():
+@pytest.mark.parametrize("name", ["k", "key"])
```

Comment:
Looks like this evolved from the [original test](https://github.com/pandas-dev/pandas/commit/ab0d23653a6e8264769b50c27b3b8efc95c6ab70#diff-50320110eccf066ad07413f479ba3294c28d1e310c089334522febb9298564f6R1499-R1502). I don't think we need to be testing both.

Reply:
Ah cool. Removed